src/vm/mod.rs

```diff
@@ -639,11 +671,22 @@ impl super::VM for CoreVM {
         value: RunExitResult,
         retry_policy: RetryPolicy,
     ) -> Result<AsyncResultHandle, Error> {
+        match &value {
+            RunExitResult::Success(_) => {
+                invocation_debug_logs!(self, "Storing side effect completed with success");
```

Comment:
Storing side effect --> journaling result
We don't use the term side effect anymore anywhere. 
Should we add the name of the run block? If we have access to that here?

Reply:
Same for the two below